select.go

```diff
@@ -193,6 +195,15 @@ func (s *Select) Run() (int, string, error) {
 	return s.innerRun(0, ' ')
 }
 
+// Runfunc in addition to Run(), it return Do func(), error.
```

Comment:
I think this should be `RunFunc` and there's a typo in the docs (_it return_ -> _it returns_)